assets/shared/data/schedule-controls.test.js

```diff
@@ -0,0 +1,31 @@
+import { registerStore } from '@wordpress/data';
+import controls, { schedule } from './schedule-controls';
+
+const delayedAction = () => ( { type: 'TEST_SCHEDULED_ACTION' } );
+
+describe( 'schedule-controls', () => {
```

Comment:
The `clearSchedule` functionality remains to be tested in a future iteration, as for some reason it is not triggering the control in a testing environment, but works in the real code.

Reply:
Should we add a card or issue to investigate that? If there is not yet.